applications/portal/backend/api/migrations/0012_remove_antibodysearch_citations_and_more.py

```diff
@@ -0,0 +1,91 @@
+# Generated by Django 4.2.9 on 2024-04-22 10:15
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0011_remove_antibodysearch_citations_and_more'),
+    ]
+
+    operations = [
+        # First need to drop the MATERIALIZED VIEW antibody_search;
+        migrations.RunSQL(
```

Comment:
@filippomc -  NOTE on The new migration file: it does the following:
1. first drops the materialized view 
2. then alters the status for antibody and vendordomain. 
3. builds the materialized view again.


this is since `cannot alter type of a column used by a view or rule`

Reply:
Thanks, good catch. Were did you take the materialized view code from?